task-sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -1215,6 +1217,10 @@ def _handle_request(self, msg: ToSupervisor, log: FilteringBoundLogger):
             )
         elif isinstance(msg, DeleteVariable):
             resp = self.client.variables.delete(msg.key)
+        elif isinstance(msg, ValidateInletsAndOutlets):
```

Comment:
Could you add a test under: `test_handle_requests` in test_supervisor.py?

Reply:
sure